### PR TITLE
Update docstring of infobox-related public functions

### LINKED ISSUE #52: Throws error when infobox not found
This is a bug introduced since PR #47. Compare:

Old functionality (03735491463ea3fd95e9885fd982c4edc9e3ad78):
```
>>> from wikipediabase.util import get_infobox, get_meta_infobox
>>> get_meta_infobox('Template:Infobox pan american games saint vincent and the grenadines').rendered_keys()
{}
```
New functionality (2c9ad51fddb5aa55aa115e5e1080d14deda003b7):
```
>>> from wikipediabase.util import get_infobox, get_meta_infobox
>>> get_meta_infobox('Template:Infobox pan american games saint vincent and the grenadines').rendered_keys()
/scratch/msilver/.virtualenvs/wikimap/local/lib/python2.7/site-packages/requests-2.7.0-py2.7.egg/requests/packages/urllib3/util/ssl_.py:90: InsecurePlatformWarning: A true SSLContext object is not available. This prevents urllib3 from configuring SSL appropriately and may cause certain SSL connections to fail. For more information, see https://urllib3.readthedocs.org/en/latest/security.html#insecureplatformwarning.
  InsecurePlatformWarning
/scratch/msilver/.virtualenvs/wikimap/local/lib/python2.7/site-packages/requests-2.7.0-py2.7.egg/requests/packages/urllib3/util/ssl_.py:90: InsecurePlatformWarning: A true SSLContext object is not available. This prevents urllib3 from configuring SSL appropriately and may cause certain SSL connections to fail. For more information, see https://urllib3.readthedocs.org/en/latest/security.html#insecureplatformwarning.
  InsecurePlatformWarning
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "build/bdist.linux-x86_64/egg/wikipediabase/util.py", line 89, in get_meta_infobox
  File "build/bdist.linux-x86_64/egg/wikipediabase/util.py", line 175, in _context_get
  File "build/bdist.linux-x86_64/egg/wikipediabase/infobox_scraper.py", line 60, in __init__
  File "build/bdist.linux-x86_64/egg/wikipediabase/infobox_scraper.py", line 126, in markup_source
  File "build/bdist.linux-x86_64/egg/wikipediabase/infobox_scraper.py", line 112, in attributes
  File "build/bdist.linux-x86_64/egg/wikipediabase/article.py", line 79, in html_source
  File "build/bdist.linux-x86_64/egg/wikipediabase/fetcher.py", line 115, in download
  File "build/bdist.linux-x86_64/egg/wikipediabase/fetcher.py", line 97, in _caching_fetch
  File "build/bdist.linux-x86_64/egg/wikipediabase/fetcher.py", line 62, in download
  File "build/bdist.linux-x86_64/egg/wikipediabase/fetcher.py", line 50, in urlopen
LookupError: Error fetching: https://en.wikipedia.org/w/index.php?action=view&redirect=yes&title=Template%3AInfobox+pan+american+games+saint+vincent+and+the+grenadines. Status code 404 : Not Found
```
This is important because the old functionality was quiet and didn't crash programs (such as WikiMap), whereas the new functionality will crash a program, and is not backwards compatible. Is it possible to achieve the good, old quiet functionality when it can't find a template?

### PR DESCRIPTION
**Pull request by alvaromorales**
Updated documentation with information about exceptions.

lxml-related errors described in #52 were fixed in a previous PR.

Fixes #52

### CODE REVIEW

File: wikipediabase/infobox.py [hunk 1]
```diff
@@ -1047,25 +1047,39 @@ def clean_attribute(attr):
 _META_INFOBOX_CACHE = util.LRUCache(10000)
 
 
-def get_infoboxes(symbol, cls=None):
+def get_infoboxes(title, cls=None):
+    """
+    Gets (markup-only) infoboxes.
```
Comment (michaelsilver) [inline wikipediabase/infobox.py]: What is the returned data format?

File: wikipediabase/infobox.py [hunk 2]
```diff
@@ -1047,25 +1047,39 @@ def clean_attribute(attr):
...
 
     if cls:
         return filter(lambda i: i.cls == cls.lower(), infoboxes)
     return infoboxes
 
 
-def get_rendered_infoboxes(symbol, cls=None):
+def get_rendered_infoboxes(title, cls=None):
+    """
+    Gets rendered infoboxes, with both markup and HTML values.
```
Comment (michaelsilver) [inline wikipediabase/infobox.py]: What is the returned data format?

**Comment by alvaromorales** [comment]
Thanks, I updated the docstrings with return info.

**Comment by michaelsilver** [comment]
LGTM. Yay for clear documentation 🎉

**Pull request merged**